src/renderer/components/NewProjectModal/NewProjectView.tsx

```diff
@@ -45,24 +46,36 @@ const NewProjectView = ({
   setProjectName,
 }: Props) => {
   const dispatch = useDispatch();
+  const [isAwaitingProjectName, setIsAwaitingProjectName] =
+    useState<boolean>(true);
 
-  const setProjectInStore = async (project: RuntimeProject) => {
-    dispatch(projectCreated(project));
-  };
+  const setProjectInStore = useCallback(
+    async (project: RuntimeProject) => {
+      dispatch(projectCreated(project));
+    },
+    [dispatch]
+  );
 
-  const handleContinue = async () => {
+  const handleContinue = useCallback(async () => {
     const project = await makeProjectWithoutMedia(projectName);
     if (project === null) {
       return;
     }
     setProjectInStore(project);
     nextView();
-  };
+  }, [nextView, projectName, setProjectInStore]);
+
+  useKeypress(handleContinue, isAwaitingProjectName, ['Enter', 'NumpadEnter']);
 
   const handleProjectNameInput = (
     event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
   ) => {
-    setProjectName(event.target.value);
+    setProjectName(event.target.value.trim());
+    if (event.target.value !== '') {
```

Comment:
Can this be simplified to something like `setIsAwaitingProjectName(event.target.value === '')` ?

Reply:
Nice catch: [c3a0de8](https://github.com/chloebrett/mlvet/pull/175/commits/c3a0de8b8c96366a77027e4b8d0f6624ec3897da)